src/vanilla/atom.ts

```diff
@@ -92,7 +92,11 @@ export function atom<Value, Args extends unknown[], Result>(
 ) {
   const key = `atom${++keyCount}`
   const config = {
-    toString: () => key,
+    toString() {
+      return import.meta.env?.MODE !== 'production' && this.debugLabel
+        ? key + ':' + this.debugLabel
```

Comment:
Curious - why add a key with `:` here if a `debugLabel` is present? 🤔 Is it to avoid naming collisions?

Reply:
Could it be to make similar with [useAtomsDevtools.ts#L12](https://github.com/jotaijs/jotai-devtools/blob/main/src/utils/useAtomsDevtools.ts#L12)?

https://github.com/jotaijs/jotai-devtools/pull/156#discussion_r1677231822

If we are going to centralize / change logic for [useAtomsDevtools.ts#L12](https://github.com/jotaijs/jotai-devtools/blob/main/src/utils/useAtomsDevtools.ts#L12) anyways, then does it make sense to consider a different delimiter instead?

Reply:
@arjunvegda toString must return a unique string as the use case is `key={}` in React. debugLabel doesn't have to be unique. So, yes, it's to avoid collisions.